pkg/controllers/streaming/processor_controller.go

```diff
@@ -312,9 +312,9 @@ func triggers(proc *streamingv1alpha1.Processor) []kedav1alpha1.ScaleTriggers {
 	for i, topic := range proc.Status.InputAddresses {
 		result[i].Type = "liiklus"
 		result[i].Metadata = map[string]string{
-			"address": strings.Split(topic, "/")[0],
+			"address": strings.SplitN(topic, "/", 2)[0],
```

Comment:
nit: not strictly needed here, right?

Reply:
No, but: more consistent and I guess, more efficient.